index.html

```diff
@@ -671,10 +671,241 @@
+						be able to deal more intelligently with collections of resources like <a>Web Publications</a>.
+
+						<p>One instance is the definition of <a href="https://drafts.csswg.org/css-content/#cross-references">cross references</a>
+						which are currently restricted to work only within a single document.
+						This restriction should be relaxed to allow for cross references between
+						separate resources of a single <a>Web Publication</a>.
+
+						<p>Another related one would be to allow <a href="https://drafts.csswg.org/css-lists-3/#counter-properties">counters</a>
+						to accumulate across multiple resources of a single <a>Web Publication</a>,
+						for instance so that figures in multiple sections may be numbered in a single sequance.
+
+					</div>
+
+				<section id="scrolling-or-paginating" class=informative>
+					<h4>Scrolling or Paginating</h4>
+					<p>Web content, and a fortiori <a>Web Publications</a>, may be rendered on various media,
+					each with differing characteristics,
+					either due to the physical media itself or to the design of the User Agent.
+					Web Publications are not constrained to any specific medium;
+					User Agents are expected to render them as they would render other web content on the same medium.
+					Authors can adjust the way they style their documents using the features specified in [[MEDIAQ]] or its successor [[MEDIAQUERIES-4]],
+					as supported by the User Agent.</p>
+					<p>It is however worth calling out a major subdivision within media types: scrolling and paginating.</p>
+					<p>In scrolling media, when content overflows the <a href="https://www.w3.org/TR/css-display-3/#initial-containing-block">initial containing block</a> (See [[CSS-DISPLAY-3]])
+					it is exposed by allowing users to scroll to it.
+					This behavior is typical of most desktop or mobile web browsers.</p>
+					<p>In paginating media, content is broken up into discrete pages,
+					and content that overflows one page is displayed on subsequent pages.
+					This behavior is typical of web content printed on paper,
+					of the output of web-to-PDF renderers,
+					and of many e-book reading systems.</p>
+					<p>Some User Agents may support both scrolling and pagination.
+					As the preferences of individual readers vary,
+					and as different types of publications may be better suited for one or the other,
+					this specification encourages User Agents to support both,
+					and to offer a choice to their users.</p>
+
+					<div class="note">
+					<p>Authors or Publishers occasionally have strong preferences between scrolling and paginating for particular types of content,
+					and may wish to require a publication to always be rendered in one or the other mode.
+					This wish is not, however, something that <a>Web Publication</a> User Agents can effectively enforce.
+					Since <a>Web Publications</a> are composed of backwards-compatible ordinary web content,
```

Comment:
I disagree and this is *very* important aspect.  it also needs to be an attribute of a publication as a given author may have a clear position on how they wish their content to be presented

Reply:
Yes, this flag / attribute / css directive should exist. Authors should express their preference and intent.

However, it can only ever be a preference. I do not see how we can possibly come up with a solution that is compatible with legacy HTML UAs (also known as browsers) and at the same time have a mandatory flags that demands that the document is presented in a paginated way. Browsers do not support it today, so effectively, it is already optional. Just the same, even if an author says they want their document to be scrolled, not paged, if you print it, it won't scroll.

In addition to the matter of capabilities, it's a User Agent, not an Author Agent, so if the User and the Author disagree, the User ultimately wins. There are things like user stylesheets, and there will be extensions, and such things can let the user override the author's preference. 

Unless we go about making documents that do not open in browsers, and documents that can be forbidden from printing, making a paged vs scroll flag mandatory to respect by UAs is just impossible.

Reply:
I'm wary of a flag or other information in a "manifest" that expresses rendering preferences. The publication author gets to write the CSS for the publication, and can express their preferences there.

Reply:
Right. I think this should be expressed in CSS (through `@page` or `@viewport` or some such, as discussed in the issue below this note). We may also want to have a way to express this for the entire collection, but that should be something that works with CSS. Maybe a way to link to a stylesheet to be shared between all resources in the collection.